.github/workflows/publish.yml

```
@@ -0,0 +1,62 @@
+name: Build and upload to PyPI
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+      # We will need cibuildwheel if we add a cython extension for common beam calcs
+      # - name: Install cibuildwheel
+      #  run: python -m pip install cibuildwheel
+      - name: Build wheels
+        run: python -m pip wheel --wheel-dir=wheelhouse .
+        # run: python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_BUILD: cp36-* cp37-* cp38-*
+          CIBW_TEST_EXTRAS: test
+          CIBW_TEST_COMMAND: pytest --pyargs radio_beam
+      - uses: actions/upload-artifact@v2
+        with:
+          path: ./wheelhouse/*.whl
+
+  build_sdist:
```

The wheel can easily be built in this section:

```suggestion
  build_sdist_and_wheel:
```